lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -429,12 +429,14 @@ public static function get_user_data_from_wp_global_styles( $theme, $create_post
 		$user_cpt         = array();
 		$post_type_filter = 'wp_global_styles';
 		$stylesheet       = $theme->get_stylesheet();
+		$post_name        = "wp-global-styles-{urlencode($stylesheet)}";
 		$args             = array(
 			'posts_per_page'         => 1,
 			'orderby'                => 'date',
-			'order'                  => 'desc',
+			'order'                  => 'asc',
 			'post_type'              => $post_type_filter,
 			'post_status'            => $post_status_filter,
+			'post_name'              => $post_name,
```

Comment:
Why is this needed? There should only ever be one global style post with the taxonomy term of the stylesheet slug.

Reply:
See this comment: https://github.com/WordPress/gutenberg/pull/47075#discussion_r1071983172